cmd/generator/main.go

```diff
@@ -107,6 +107,7 @@ var generatorCmd = &cobra.Command{
 			"mattermost-plugin-jenkins": "data/icons/jenkins.svg",
 			"mattermost-plugin-jira":    "data/icons/jira.svg",
 			"mattermost-plugin-webex":   "data/icons/webex.svg",
+			"mattermost-plugin-zoom":    "data/icons/zoom.svg",
```

Comment:
Hosting the images locally was just meant as a stop-gap until the plugins defined the icons themselves, and posted a release containing same. Could we just wait for that, and slowly reduce this list to zero instead?

Reply:
Given that there are no "old" release of the zoom plugin that need the icon, I agree that it's not worth continuing to use this workaround. I will close the PR.